pkgs/development/compilers/ocaml/3.11.2.nix

```diff
@@ -69,6 +69,7 @@ stdenv.mkDerivation rec {
        '';
 
     platforms = with platforms; linux ++ darwin;
+    hydraPlatforms = with platforms; linux ++ darwin;
```

Comment:
No, I do not think so, the release script fallbacks on platforms if there is no hydraPlatforms attribute.